tests/level1_test.go

```diff
@@ -0,0 +1,27 @@
+package caf_tests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFoundationResourceGroupsDoesNotExist(t *testing.T) {
+	t.Parallel()
+
+	test := prepareTestTable()
+
+	client, _ := azure.GetResourceGroupClientE(test.SubscriptionID)
+
+	var top int32 = 100
+
+	result, _ := client.List(context.Background(), "tagName eq 'level' and tagValue eq 'level1'", &top)
```

Comment:
This failed in my subscription since I have 2 deployments, need to keep the env check as well.

Reply:
Actually @polatengin , I don't believe this test verifies foundation. Assuming that you are deploying the demo configuration, the resource group tagged with level=level1 is created via launchpad.

I think in this case, we should add as single foundation resource group tagged with level=leve1 and key=foundation. Something similar to this.  I can work with you on setting up the configs needed.
https://github.com/Azure/caf-terraform-landingzones/blob/master/landingzones/caf_eslz/scenario/100/enterprise_scale_contoso_com.tfvars#L1

We should also add comments that this is a no-op test and we're placing the tests there in order to provide a place to include future foundation level tests as needed for the specific symphony project.

Reply:
thank you for your feedback. I just updated the tests to take the value of the environment tag into consideration. if you have more than one environment with CAF, only the given environment is under test